SPECS/perftest/perftest.spec

```diff
@@ -0,0 +1,80 @@
+Summary:        IB Performance tests
+Name:           perftest
+# Update extended_release with version updates
+Version:        4.5
+Release:        1%{?dist}
+License:        BSD 3-Clause, GPLv2
```

Comment:
We should use `and` and `or` as separators, to make the licensing situation  unambiguous.

Reply:
Also, `BSD 3-Clause` can be shortened to `BSD`.